### PR TITLE
Priya S

### PR DESCRIPTION
**Pull request by ps975076**

### CODE REVIEW

File: assets/img/HTML Spotify Challenge_files/style.css
Comment (Kocibey01) [inline assets/img/HTML Spotify Challenge_files/style.css:1]: Very clear work. Well done Priya 💯

Comment (Kocibey01) [inline assets/img/HTML Spotify Challenge_files/style.css:1]: It's very clear and readable. Great job Priya! 👍

File: index.html [hunk 1]
```diff
@@ -1,12 +1,336 @@
...
+            </div>
+          </li>
+        </ul>
+      </div>
+
+      <hr class="createandliked" />
+
+      <!-- Playlists -->
+      <div class="list-of-playlists">
+        <div class="Bhangra-Party-Mix"><p>Bhangra Party Mix</p></div>
```
Comment (ManNavic) [inline index.html:115]: I think you could have used one class for all the playlists here

File: index.html [hunk 2]
```diff
@@ -1,12 +1,336 @@
...
+            <img src="./assets/img/einaudi.jpeg" width="250px" />
+          </div>
+          <div class="album-name10"><p>Einaudi</p></div>
+        </div>
+        <!-- Row 3 ending Div -->
+      </section>
+
+      <!-- .................................... -->
+
+      <!-- Row 4 - Bottom-Row (Controls) -->
```
Comment (ManNavic) [inline index.html:315]: Nice commenting easy to read 👍

File: assets/img/HTML Spotify Challenge_files/style.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  right: 0;
+  width: 2300px;
+  background-color: yellowgreen;
+  padding: 20px;
+  height: 150px;
+}
+
+.hello {
+  color: blue;
+}
```
Comment (ManNavic) [inline assets/img/HTML Spotify Challenge_files/style.css:138]: Great Job Priya :)

File: assets/img/HTML Spotify Challenge_files/style.css
Comment (ps975076) [reply assets/img/HTML Spotify Challenge_files/style.css:1]: Thanks Koci!

**Review by Kocibey01 — COMMENTED** [review]
Comments on the HTML and CSS pages are very useful. 👍

File: assets/img/HTML Spotify Challenge_files/style.css (hunk 3 above)
Comment (ps975076) [reply assets/img/HTML Spotify Challenge_files/style.css:138]: Cheers Mantas

**Pull request closed without merging**